vm-setup/roles/libvirt/tasks/network_teardown_tasks.yml

```diff
@@ -23,6 +23,12 @@
 
   when:
     - ansible_distribution == 'Ubuntu'
-
   become: yes
 
+- name: Disable the firewall forward rule
+  iptables:
+    chain: FORWARD
+    state: absent
+  when:
+    - EPHEMERAL_CLUSTER == 'kind'
```

Comment:
I don't think we should restrict the solution to Kind only

Reply:
As this reject rule is appearing when you have kind cluster, I used this condition.  do you think that this condition doesn't need at all.

Reply:
The problem was kind specific but we can make the solution to work both for kind and minikube.  So you can just remove the lines 32-33

Reply:
In that case I don't need to use any condition.

Reply:
Yes. We can remove it.

Reply:
Done